nodepool/config/templates/stack-template.json

```diff
@@ -73,7 +73,8 @@
                   {{if gt $spec.RootVolumeIOPS 0}}
                   "Iops": "{{$spec.RootVolumeIOPS}}",
                   {{end}}
-                  "VolumeType": "{{$spec.RootVolumeType}}"
+                  "VolumeType": "{{$spec.RootVolumeType}}",
+                  "Encrypted": {{$spec.RootVolumeEncrypted}}
```

Comment:
Thanks for the PR! :bow:
Any chance you could fix somewhere so that the travis build passes?
I suspect this line as the source of the problem. If I understand correctly, shouldn't it be `WorkerRootVolumeEncrypted` rather than `RootVolumeEncrypted`?